applications/StructuralMechanicsApplication/tests/test_axis_projection.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import print_function, absolute_import, division
+import KratosMultiphysics
+
+import KratosMultiphysics.StructuralMechanicsApplication as StructuralMechanicsApplication
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import math
+
+class TestAxisProjection(KratosUnittest.TestCase):
+
+    def _set_up_test_case(self):
+        current_model = KratosMultiphysics.Model()
+        mp = current_model.CreateModelPart("solid_part")
+        #create cube
+        mp.CreateNewNode(1,0.0,0.0,0.0)
+        mp.CreateNewNode(2,2.0,0.0,0.0)
+        mp.CreateNewNode(3,2.0,2.0,0.0)
+        mp.CreateNewNode(4,0.0,2.0,0.0)
+
+        mp.CreateNewNode(5,0.0,0.0,3.0)
+        mp.CreateNewNode(6,2.0,0.0,3.0)
+        mp.CreateNewNode(7,2.0,2.0,3.0)
+        mp.CreateNewNode(8,0.0,2.0,3.0)
+
+        element_name = "ShellThinElementCorotational3D4N"
```

Comment:
no test with membrane?

Reply:
the membrane element is not in the master ye